git_machete/client.py

```diff
@@ -1945,12 +1944,12 @@ def __sync_before_creating_pr(self) -> None:
             if s == BEHIND_REMOTE:
                 warn(f"Branch {current_branch} is in <b>BEHIND_REMOTE</b> state.\nConsider using 'git pull'.\n")
                 self.__print_new_line(False)
-                ans = self.ask_if("Proceed with pull request creation?" + get_pretty_choices('y', 'q'),
+                ans = self.ask_if("Proceed with pull request creation?" + get_pretty_choices('y', 'Q'),
```

Comment:
Hmmm again, as with one of the previous cases... what happens if someone selects `n` or generally something else then `y` or `q` here? 🤔